docker/README.md

```diff
@@ -62,12 +62,90 @@ docker-compose -f docker-compose-mysql.yml down
 
 Please note that SQL support is still in active developement and it is not production ready yet.
 
-Quickstart for production
+One host production setup with docker-compose
 =========================
-In a typical production setting, dependencies (cassandra / statsd server) are
-managed / started independently of the cadence-server. To use the container in
-a production setting, use the following command:
+For a simple production setup, start with our **docker-compose-production-example.yml** for a single host Cadence cluster as described in the following steps.
 
+#### 1. Modify the database mounting path in the example YAML file
+You may want to set the Cassandra data location because:
+* a. You need a disk partition that has enough space
+* b. You prefer to have separated storage hardware for have a separated storage hardware for database for performance, backup plan, and availability (with RAID)
+* c. Setting a mounting path can avoid losing data when you run **docker-compose down**
+
+Save the edited example YAML file as docker-compose.yaml.
+
+See the [Docker Documentation](https://docs.docker.com/compose/production/) for more information about using docker-compose in production.
+
+#### 2. Run docker-compose
+```
+docker-compose up -d
+```
+Try out our [samples](https://github.com/uber-common/cadence-samples) to make sure the server is running well.
+
+#### 3. Upgrade Cadence server
+* a. First, you need to upgrade the database schema. You can find instructions for using the cadence-cassandra-tool to upgrade schema [here](https://github.com/uber/cadence/tree/master/tools/cassandra#update-schema-as-part-of-a-release).
+* b. Shut down docker-compose by using the following command:
+```
+docker-compose stop
+```
+* c. Modify the version tag in Cadence server to the new release in docker-compose.yaml. Then run it again:
+```
+docker-compose up -d
+```
+
+#### 4. Backup & Restore
+* **Backup**: Use nodetool [snapshot](http://cassandra.apache.org/doc/latest/tools/nodetool/snapshot.html) command to create hot snapshots.
+ To access nodetool, you can install nodetool, or log on to the Cassandra container. Get your Cassandra docker container ID by:
+ ```
+ docker ps
```

Comment:
Put a `$` in front of the command so that it's easier to distinguish the command from its output.

Reply:
Done